python/lsst/pipe/tasks/multiBand.py

```diff
@@ -99,6 +99,11 @@ def getShortFilterName(name):
 ##############################################################################################################
 
 class DetectCoaddSourcesConfig(Config):
+    """!
+\anchor DetectCoaddSourcesConfig
```

Comment:
(Throughout) is it necessary to have the documentation out-dented to the leftmost column? It doesn't seem to be the norm in other tasks.